relay/blockchain/delegate.py

```diff
@@ -6,6 +8,10 @@
     deploy_proxied_identity,
 )
 
+META_TRANSACTION_FEE_VALUE = 1
```

Comment:
Am I correct that this fixed value solution becomes them adjusted by #351 ?

Reply:
No, the current user story suggests "For now, we use a flat-flat fee (any meta transaction will be relayed for x - where is can be set)". I would not know right now how to use a smarter value than this.

Reply:
I could make this value adjustable via the config of the relay server but I do not believe it makes much more sense.

Reply:
So what will #351 be then about? Check if the send fee amount is <= this variable?

Reply:
>=, but yes.

Reply:
Haha, sure. Thanks for your help.